spryker/usr/local/share/container/baseimage-45.sh

```diff
@@ -4,11 +4,15 @@ source /usr/local/share/spryker/spryker_functions.sh
 alias_function do_build_permissions do_spryker_build_permissions_inner
 do_build_permissions() {
   do_spryker_build_permissions_inner
-  do_spryker_build
+  do_spryker_directory_create
+  do_spryker_config_create
 }
 
 alias_function do_build do_spryker_nginx_build_inner
 do_build() {
+  export DEVELOPMENT_MODE="${BUILD_DEVELOPMENT_MODE}"
+  export APPLICATION_ENV="${BUILD_APPLICATION_ENV}"
```

Comment:
BUILD_* is obsolete and never needed, projects can just set DEVELOPMENT_MODE and APPLICATION_ENV